src/quest_manager/models.py

```diff
@@ -340,17 +355,18 @@ def get_queryset(self, include_archived=False):
         return qs
 
     def get_active(self):
-        return self.get_queryset().datetime_available().not_expired().visible()
+        return self.get_queryset().datetime_available().not_expired().visible().active_or_no_campaign()
 
     def get_available(self, user, remove_hidden=True, blocking=True):
         """ Quests that should appear in the user's Available quests tab.   Should exclude:
         1. Quests whose available date & time has not past, or quest that have expired
         2. Quests that are not visible to students or archived
-        3. Quests who's prerequisites have not been met
-        4. Quests that are not currently submitted for approval or already in progress
-        5. Quests who's maximum repeats have been completed
-        6. Quests who's repeat time has not passed since last completion
-        7. Check for blocking quests (available and in-progress), if present, remove all others
+        3. Quests that are a part of an inactive campaign
+        4. Quests whose prerequisites have not been met
+        5. Quests that are not currently submitted for approval or already in progress
+        6. Quests whose maximum repeats have been completed
+        7. Quests whose repeat time has not passed since last completion
+        8. Check for blocking quests (available and in-progress), if present, remove all others
         """
         qs = self.get_active().select_related('campaign')  # exclusions 1 & 2
         qs = qs.get_conditions_met(user)  # 3
```

Comment:
The code comment references to the docstring seem to be off now.  Comment says # 3, but it's # 4 in your new docstring list.

Reply:
code comments updated to reflect new numbering.
```
        qs = self.get_active().select_related('campaign')  # exclusions 1, 2 & 3
        qs = qs.get_conditions_met(user)  # 4
        available_quests = qs.not_submitted_or_inprogress(user)  # 5, 6 & 7

        available_quests = available_quests.block_if_needed(user=user)  # 8
```